src/app.ts

```diff
@@ -113,6 +114,18 @@ export async function getApp(configOverrides: ConfigOverrides = {}) {
     },
   })
 
+  await app.register(fastifyCors, {
+    origin: '*',
```

Comment:
in production we probably don't want to accept all origins, I would do a switch between prod and dev here